### ISSUE TITLE
Add time frame as URL parameter

### ISSUE THREAD

**Issue by blacktemplar** [ultrasoundmoney/frontend#338]
Adding the selected time frame as URL parameter has multiple advantages:

1. Time frame gets remembered when refreshing the page
2. One can share links for a specific time frame

This should of course happen without refreshing the page when changing the time frame, just update the URL. This should be possible technically.

**Comment by alextes** [comment]
i like this idea. we could have multiple parameters as there are multiple dashboards with time frames. it would work even better when we finally split the page into multiple pages / sections so each page can have its own straightforward time frame parameter.

**Comment by Rahul-K-Saini** [comment]
Hi, @alextes
Can i work on this, i am thinking of adding the time frame, currency and pow to URL so the state remain persistence even after reloading or sharing the URL. I have experience working with features like this

**Comment by alextes** [comment]
@Rahul-K-Saini of course. i implemented a proof-of-concept a long time ago myself. i think at the time i started seeing annoying server side rendering issues but i think that was because i stored the last set time frame in local storage causing a mismatch. if it's put in the URL the server could render the correct page right away.

either way, just make sure to test refresh and hop between time frames please but i think with the URL approach should be fine.

**Comment by Rahul-K-Saini** [comment]
@alextes on it.

**Comment by Rahul-K-Saini** [comment]
let me check

**Comment by alextes** [comment]
sure thank you! if you can fix it in an hour great, but no need to spend a lot of time on it.

update is already live btw 🚀
